tests/config/vitest.config.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Vitest config file
+ * @see https://vitest.dev/config/
+ */
+
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+	test: {
+		environment: 'jsdom',
+		include: [
+			'tests/unit/**/*.test.ts'
+		],
+		setupFiles: [
+			'tests/config/vitest.setup.ts'
```

Comment:
Leaving this here so we can quickly use it should we need it sometime later?

Reply:
I think so — let's merge this for now, and then do a more thorough review when comparing all plugins with e2e tests.